connector/launchers/runtime-embedded/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Stage 1: Build the JAR using Gradle
+FROM eclipse-temurin:23.0.2_7-jdk-alpine AS builder
+
+WORKDIR /app
+
+# Copy the Gradle configuration files
+COPY ./gradlew ./gradlew
+COPY ./gradle ./gradle
+COPY ./build.gradle.kts ./build.gradle.kts
+COPY ./gradle.properties ./gradle.properties
+COPY ./settings.gradle.kts ./settings.gradle.kts
```

Comment:
Since all files will not be renamed, we can use this syntax instead `COPY file1 file2 file3 destination_dir`
You'll end up with less layers in the Docker image, thus, lower image size